.github/workflows/ingest.yaml

```diff
@@ -0,0 +1,80 @@
+name: Ingest
+
+defaults:
+  run:
+    # This is the same as GitHub Action's `bash` keyword as of 20 June 2023:
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
+    #
+    # Completely spelling it out here so that GitHub can't change it out from under us
+    # and we don't have to refer to the docs to know the expected behavior.
+    shell: bash --noprofile --norc -eo pipefail {0}
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        description: 'Specific container image to use for ingest workflow (will override the default of "nextstrain build")'
+        required: false
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      image:
+        description: 'Specific container image to use for ingest workflow (will override the default of "nextstrain build")'
+        required: false
+        type: string
+      trial_name:
+        description: |
+          Trial name for outputs.
+          If not set, outputs will overwrite files at s3://nextstrain-data/files/workflows/lassa/
+          If set, outputs will be uploaded to s3://nextstrain-data/files/workflows/lassa/trials/<trial_name>/
+        required: false
+        type: string
```

Comment:
Copying my comments from [rabies](https://github.com/nextstrain/rabies/pull/2#discussion_r1676548216) because I heard @j23414 mention lassa automation in the priorities meeting

> Noting that the this GH Action workflow will error out until rabies is added to the nextstrain/infra repo. See [docs for automation](https://github.com/nextstrain/pathogen-repo-guide/blob/main/docs/nextstrain-internal/automated-pathogen-workflows.md).

> I want to be explicitly clear here that this workflow does not have an automated schedule. Zika's automated workflow ([ingest-to-phylogenetic](https://github.com/nextstrain/zika/blob/573ebd7d6c6f58e26b5f02f9c7ef2473623daab2/.github/workflows/ingest-to-phylogenetic.yaml#L32)) calls on the ingest workflow and phylogenetic workflow. Once the phylogenetic workflow is set up in this repo, they can be connected the same way.
> 
> Not asking for any changes in this PR, just wanted to make sure we're all on the same page 😄

Reply:
Adding lassa to supported pathogen repos in https://github.com/nextstrain/infra/pull/26